packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -327,7 +335,47 @@ export class Coachmark extends BaseComponent<ICoachmarkProps, ICoachmarkState> {
   }
 
   public componentWillUnmount(): void {
+    const { preventDismissOnLostFocus } = this.props;
+
     this._events.off(document, 'keydown', this._onKeyDown, true);
```

Comment:
the events will be unmounted for you on unmount, so you don't need to do this manuall.

Reply:
you can also say `this._events.off()` to remove all events.